### ISSUE TITLE
Doesn't download any image

### ISSUE THREAD

**Issue by 0x48piraj** [SubhrajitPrusty/unsplash-download#5]

**Comment by 0x48piraj** [comment]
Doesn't downloads, you didn't fixed it.

**Comment by SubhrajitPrusty** [comment]
Works for me. Works for my peers. On both windows and Linux.

Program successfully gets the images. wget seems to fail for you. 
Problem is on your end.

Check your internet connection. Check if you are getting blocked by a firewall.

**Comment by 0x48piraj** [comment]
Very cool answer. Thanks.

**Comment by SubhrajitPrusty** [comment]
> Very cool answer. Thanks.

Being sarcastic won't fix your problem. 
There's nothing wrong with the program.

If you could do a simple google search you would get [this](https://superuser.com/questions/1182465/wget-returning-error-unable-to-establish-ssl-connection#1182481), which describes a similar problem.
And the required solution.

As you've stated before, you can use requests to download the images. But I won't add that to my code since I still think wget is way better to use.
